proposals/exception-handling/Exceptions-formal-overview.md

```diff
@@ -152,34 +154,34 @@ An absent tag address in a `catch` administrative instruction (i.e., `a? = ε`)
 ```
 F; throw x  ↪  F; throw a  (if F.module.tagaddrs[x]=a)
 
-caught{a val^n} B^l[rethrow l] end
-  ↪ caught{a val^n} B^l[val^n (throw a)] end
+caught{a val*} B^l[rethrow l] end
```

Comment:
In the formal spec, when do we use `^n` and when do we use `*`? Can they be used interchangeably? I see both notations in the current (standardized) formal spec and I'm not very familiar with the formal notation, so..

Reply:
IIUC we use ^n when the n has a constraint, but I think in the formal spec document this is not always the case.

This change is to implement exactly this idea, so to only use the n if it has a particular constraint, as it does for example in lines 163-164.

I think that this change makes the formal overview a bit more concise, as I see the formal overview as a "cheatsheet" for Wasm exceptions.